client/state/stored-cards/test/reducer.js

```diff
@@ -20,8 +20,13 @@ import {
 } from 'state/action-types';
 import reducer, { items } from '../reducer';
 import { STORED_CARDS_FROM_API } from './fixture';
+import { useSandbox } from 'test/helpers/use-sinon';
 
 describe( 'items', () => {
+	useSandbox( ( sandbox ) => {
+		sandbox.stub( console, 'warn' );
```

Comment:
We expect a warning in one of our tests. If we didn't stub, we'd see the warning output while running the test.

eg compare the output of the following with and without the stub:
```
npm run test-client client/state/stored-cards/test/reducer.js
```